src/Engine/ProtoCore/DSASM/Executable.cs

```diff
@@ -50,6 +50,9 @@ public enum OffsetConstants
         public ProtoCore.DSASM.ProcedureTable[] procedureTable { get; set; }
         public ProtoCore.DSASM.SymbolTable[] runtimeSymbols { get; set; }
 
+        public List<CodeBlock> CodeBlocks { get; set; }
```

Comment:
Will `CodeBlocks` be set somewhere else? If not, I'd suggest to make it a private set and use `IEnumerable<CodeBlock>` as its type. Same for `CompleteCodeBlocks`.